application.py

```diff
@@ -1,32 +1,96 @@
-from flask import Flask
 
-# print a nice greeting.
-def say_hello(username = "World"):
-    return '<p>Hello %s!</p>\n' % username
+# coding: utf-8
+
+# In[ ]:
+
+
+from flask import Flask, redirect, url_for, session, request, jsonify
+from requests_oauthlib import OAuth2Session
+from requests_oauthlib.compliance_fixes import linkedin_compliance_fix
+import os 
+
+#for SSL
+os.environ['OAUTHLIB_INSECURE_TRANSPORT'] = '1'
+
+
+# In[ ]:
+
+
 
 # some bits of text for the page.
 header_text = '''
     <html>\n<head> <title>EB Flask Test</title> </head>\n<body>'''
 instructions = '''
-    <p>Hi guys! -parth</p>\n'''
+    <p>Hi guys! -parth</p>\n
+    <p> Hello - augustus </p>
+    <p> What's up - Niral </p>
+    <a href="/authorize">login</a>
+'''
+
 home_link = '<p><a href="/">Back</a></p>\n'
 footer_text = '</body>\n</html>'
 
+
 # EB looks for an 'application' callable by default.
 application = Flask(__name__)
+application.secret_key = os.urandom(24)
+
+client_id = '78gbrnp4ovmhd3'
```

Comment:
stuff like id, secrets, token_url should probably be environment variables xD especially since this is a public repo.